content/module-reference/utility-modules/shared/metadata-editor.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: metadata editor
 id: metadata-editor
-applicable-version: 3.2.1
+applicable-version: 3.6.0
```

Comment:
are you saying that the edits you have made are to functionality introduced in 3.6? Usually we just put the version field to the next major release (so 4.0.0)